haiku-apps/burnitnow/burnitnow-1.1.recipe

```diff
@@ -25,12 +26,12 @@ REQUIRES="
 	"
 
 BUILD_REQUIRES="
-	haiku_devel
+	haiku${secondaryArchSuffix}_devel
```

Comment:
This change isn't needed, there no secondary arch for this app.

Reply:
Damned. I'm lost on when it's needed and when it's not...

At least it doesn't harm here. 
Will fix it anyway.